### ISSUE TITLE
Donation wrap MOSTWANTED

### ISSUE THREAD

**Issue by NitroZeus99** [saesrpg/donation-vehicle-shaders#246]
**Vehicle ID**
405

**Wrap access**
alex291999

**Needs testing**
yes

**Thread for reward change**

https://saesrpg.uk/topic/25995/donation-_nitrozeus_-amount-10-00-gbp?_=1616599416439

**Translations**

**Wrap image**
![template - Copia](https://user-images.githubusercontent.com/81317557/112340332-bac4c800-8cc0-11eb-99a1-c1b7257aeb33.png)

**Comment by tut98** [comment]
The wrap template will be down for maintenance.

**Comment by tut98** [comment]
Ok, it's up again - please get the latest at https://saesrpg.uk/topic/8309/about-donation-wraps
ps. it's always been facing east, so you will have to turn it 90 degrees clockwise. Nothing much changed so adapting your wrap to the new one should take little time.

**Comment by NitroZeus99** [comment]
oky let me check and in case ill open another new issue

**Comment by tut98** [comment]
It's fine if you continue using this issue.

**Comment by NitroZeus99** [comment]
what is the purpose of the helpers ? if i color over it will mess something?

**Comment by NitroZeus99** [comment]
![template123](https://user-images.githubusercontent.com/81317557/112383364-510ee300-8ced-11eb-9288-416c7e4f8332.png)

okay....i admit the new template was a bit harder bc of the less accuracy...but it should be good...i need testing again and some comments from where i missed the color (example missed blue spots)

**Comment by tut98** [comment]
A preview will be sent within 72 hours.
Helpers layer is only to know where to draw at, should not include them in final image.
Template should be more accurate than the former.

**Comment by NitroZeus99** [comment]
okay let me reupload without the helpers

**Comment by NitroZeus99** [comment]
![template](https://user-images.githubusercontent.com/81317557/112390142-237a6780-8cf6-11eb-878f-316b3781cc8a.png)
done

**Comment by tut98** [comment]
https://imgur.com/a/gdxgOqB
**Two issues**:
The roof can be painted black with 2nd color option ingame, no need to paint it on wrap.
The wrap has been misplaced on the canvas - please move it back to where it should be (check original). I've tried moving it myself but it needs tweaks to fit properly, hence artifacts showing on the screenshots.

**Comment by NitroZeus99** [comment]
the roof should stay painted black bc paintjob reasons , if you have discord we can fix it together

**Comment by tut98** [comment]
You can paint the roof black by using color option 2 at a modshop now, so it's best that you keep the roof white.

**Comment by NitroZeus99** [comment]
i'm checking the stock canvas...the problem is misplaced from the original

**Comment by tut98** [comment]
Hmm okay I'll check the template.

**Comment by NitroZeus99** [comment]
literally i have my file and the stock ...the canvas are the same

**Comment by tut98** [comment]
You might have overwritten the stock version - please re-download it [here](https://saesrpg.uk/topic/8309/about-donation-wraps)

**Comment by NitroZeus99** [comment]
i've re-downloaded it back...
same

**Comment by NitroZeus99** [comment]
can be a bit of overcolor over the template, i mean...coloring by photoshop and going over of 1 pixel can mess the mesuring?

**Comment by NitroZeus99** [comment]
okay i see it its a bit over the canvas, ill try to fix it

**Comment by tut98** [comment]
Sounds good, I've just double checked and template looks fine!

**Comment by NitroZeus99** [comment]
can be the helper draw a bit better? im having trouble with the bumpers

**Comment by NitroZeus99** [comment]
ill try to follow the helpers

**Comment by NitroZeus99** [comment]
![LASTONE](https://user-images.githubusercontent.com/81317557/112735279-2c17ab80-8f4b-11eb-9546-1eb979d5e185.png)
 try this

**Comment by NitroZeus99** [comment]
![LASTONE](https://user-images.githubusercontent.com/81317557/112735551-20c57f80-8f4d-11eb-9a79-06630fd1aac3.png)
try this....

**Comment by tut98** [comment]
I recommend you don't use white outline where there's blue - i.e the waves on the side, they should extend to the bottom of the car and not be blocked by a white outline. It will not look good.
You've had one test and I think that's sufficient.

**Comment by NitroZeus99** [comment]
okay let me change fast then upload okay? ^^
ill edit this comment with the finished product

**Comment by tut98** [comment]
Wrap's on its way as `alex291999_1`

**Comment by NitroZeus99** [comment]
<3 i hope with no problems
